consensus/rpcserver/server.go

```diff
@@ -0,0 +1,42 @@
+package consensusrpcserver
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/consensus"
+	"github.com/offchainlabs/nitro/execution"
+)
+
+type ConsensusRPCServer struct {
+	consensus consensus.FullConsensusClient
+}
+
+func NewConsensusRpcServer(consensus consensus.FullConsensusClient) *ConsensusRPCServer {
```

Comment:
nitpick: golang naming convention
```suggestion
func NewConsensusRPCServer(consensus consensus.FullConsensusClient) *ConsensusRPCServer {
```